hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -669,6 +669,14 @@
       production environments.
     </description>
   </property>
+  <property>
+    <name>ozone.om.db.dirs.permissions</name>
+    <value>750</value>
+    <description>
+      Permissions for the metadata directories for Ozone Manager The
+      permissions have to be octal.
```

Comment:
As SCM,Recon DB directories you marked it can be either octal or symbolic, I think OM directory permission can also be symbolic and not only octal.

Reply:
Thanks for the suggestion! Initially, I believed that providing octal permissions for configurations would be sufficient. However, since you mentioned the importance of supporting both octal and symbolic formats, I realized it makes sense. It will be easier for users to provide permissions in the symbolic format rather than octal. Therefore, I have made the necessary changes to support both formats and have also added unit tests to ensure everything works correctly. 

@ashishkumar50